runelite-client/src/main/java/net/runelite/client/plugins/timers/TimerTimer.java

```diff
@@ -33,9 +35,12 @@
 {
 	private final GameTimer timer;
 
+	static ItemManager itemManager;
+	static SpriteManager spriteManager;
+
 	public TimerTimer(GameTimer timer, Plugin plugin)
```

Comment:
plugin.getItemManager and plugin.getSpriteManager. And remove the static assigns. or even send both to constructor

Reply:
I went ahead with the constructor approach.